docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -1637,6 +1637,13 @@ Default stratifications to all observers should include scenario and input draw.
     - Default
     - Default
     - Default
+  * - 20.2
```

Comment:
How are you deciding to make these "minor runs" aka 20.2 rather than "major runs" and label it as 21? 

I don't think it's wrong to do this, but I also don't think we have great standard practices for this.

Reply:
It's pretty fuzzy, but I've sort of been thinking of it as "major functionality change" vs "minor/data change." I don't think I've been incredibly consistent about it, though. 20.0 probably should have been a minor version.

It is worth noting, I think this update will be completely task-shifted, as it is just updating a key in the artifact.

Reply:
this also need not prevent you from merging this PR, I just think we should decide and communicate this to everyone.